stores/bench_test.go

```diff
@@ -64,10 +63,10 @@ func BenchmarkArchiveContract(b *testing.B) {
 // BenchmarkPruneHostSectors benchmarks the performance of pruning host sectors.
 //
 // cpu: Apple M1 Max
-// BenchmarkPruneHostSectors-10                5007            243928 ns/op            1776 B/op         49 allocs/op
+// BenchmarkPruneHostSectors-10                   4         333581052 ns/op           79684 B/op       1711 allocs/op
```

Comment:
Would be nice to eventually update this to use `SetBytes` and output the number of data pruned from a contract for better readability.

Reply:
It's not exact but I'm now using `batch size` x `num_sectors` as the number of bytes processed by a single operation. It's roughly 140GB/s which is nothing to write home about but it's fast enough imo seeing that it's a background loop that's limited to 10k sectors at a time (roughly ~10ms each)